src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -5,60 +5,112 @@
             [quo2.core :as quo2]
```

Comment:
this can be updates to `:as quo`

Reply:
I think Andrey said we are using quo2 in status-im?